deploy-manage/monitor/logging-configuration/logfile-audit-output.md

```diff
@@ -8,17 +9,19 @@ applies:
   stack: all
 ---
 
-% evaluate the applies section
+# Elasticsearch logfile audit output [audit-log-output]
 
-# Logfile audit output [audit-log-output]
+The `logfile` audit output is the only output for auditing. By default, it writes data to the `<clustername>_audit.json` file in the logs directory. The file is also rotated and archived daily or upon reaching the 1GB file size limit.
```

Comment:
the logs dir is irrelevant to everything but self-managed, right?

Reply:
yes, that's one of my main concerns of this page :-D
Help!!!